File: Examples/CSharp/Articles/MoveRangeOfCells.cs
using System;
using System.IO;

using Aspose.Cells;

namespace Aspose.Cells.Examples.CSharp.Articles
{
    public class MoveRangeOfCells
    {
        public static void Run()
        {
            //Source directory
            string sourceDir = RunExamples.Get_SourceDirectory();

            //Output directory
            string outputDir = RunExamples.Get_OutputDirectory();
            
            // Open the Excel file
            Workbook workbook = new Workbook(sourceDir + "sampleMoveRangeOfCells.xlsx");

            Cells cells = workbook.Worksheets[0].Cells;

            // Create Cell's area
            CellArea ca = CellArea.CreateCellArea("A1", "C5");
            
            // Move Range
            cells.MoveRange(ca, 7, 5);

            // Save the resultant file
            workbook.Save(outputDir + "outputMoveRangeOfCells.xlsx");

            Console.WriteLine("MoveRangeOfCells executed successfully.");
        }
    }
}

File: Examples/VisualBasic/Articles/MoveRangeOfCells.vb
Imports Microsoft.VisualBasic
Imports System.IO

Imports Aspose.Cells

Namespace Articles
    Public Class MoveRangeOfCells
        Public Shared Sub Run()
            ' ExStart:1
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)
            ' Instantiate the workbook object
            ' Open the Excel file
            Dim workbook As New Workbook(dataDir & "book1.xls")

            Dim cells As Global.Aspose.Cells.Cells = workbook.Worksheets(0).Cells

            ' Create Cell' S area
            Dim ca As New CellArea()
            ca.StartColumn = 0
            ca.EndColumn = 1
            ca.StartRow = 0
            ca.EndRow = 4

            ' Move Range
            cells.MoveRange(ca, 0, 2)

            ' Save the resultant file
            workbook.Save(dataDir & "output.xls")
            ' ExEnd:1
        End Sub
    End Class
End Namespace